railties/lib/rails/forking.rb

```diff
@@ -0,0 +1 @@
+Rails.application.load_forkers
```

Comment:
Do we need this extra file? I feel like we should just call the code here directly.